msteams-platform/tutorials/code-samples.md

```diff
@@ -3,11 +3,21 @@ title: Microsoft Teams code samples
 description: Links and descriptions of sample applications for the Microsoft Teams developer platform
 keywords: Microsoft Teams developer samples
 ---
-# Code samples for the Microsoft Teams developer platform
+# Tutorials and code samples for the Microsoft Teams developer platform
 
-Here you will find a list of code samples that demonstrate various capabilities of the Microsoft Teams development platform and how to build apps to leverage those features.
+Here you will find a list of tutorials and code samples that demonstrate how you can extend the Teams developer platform capabilities by creating custom apps.
```

Comment:
Not a big fan of "developer platform" here - they aren't extending the developer platform, they are using the developer platform to extend Teams.